src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_CIRCLE_RADIUS = 10;
+    private static final int MAX_RANDOM_VALUE = 100;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int index = random.nextInt(FigureType.values().length);
+        FigureType randomType = FigureType.values()[index];
+        double firstSide = random.nextInt(MAX_RANDOM_VALUE);
+        double secondSide = random.nextInt(MAX_RANDOM_VALUE);
```

Comment:
I think we should generate random parameters while creating a figure

Reply:
Fixed